apps/mobile/src/libs/components/button/styles.ts

```diff
@@ -3,13 +3,29 @@ import { StyleSheet } from "react-native";
 import { BaseColor } from "~/libs/enums/enums";
 
 const styles = StyleSheet.create({
-	btn: {
-		alignItems: "center",
-		backgroundColor: BaseColor.BLACK,
-		padding: 10,
+	bgWhite: {
+		backgroundColor: BaseColor.BG_WHITE,
 	},
-	label: {
-		color: "white",
+	colorBlack: {
+		color: BaseColor.BLACK,
+	},
+	colorWhite: {
+		color: BaseColor.BG_WHITE,
+	},
+	p1: {
+		padding: 1,
+	},
+	rounded: {
+		borderRadius: 35,
+	},
+	text: {
+		fontFamily: "Nunito",
+		fontSize: 16,
+		fontWeight: 700,
```

Comment:
Don't set these via styles, use Text props `size` and `weight`. Also, it would be better to add `uppercase` preset to the <Text /> and use it in this case